sse2neon.h

```diff
@@ -2510,11 +2510,46 @@ FORCE_INLINE __m128 _mm_fmadd_ps(__m128 a, __m128 b, __m128 c)
 #endif
 }
 
+// Computes the fused multiple add product of 32-bit floating point numbers.
+//
+// Return Value
+// Multiplies A and B, and adds C to the temporary result before returning it.
```

Comment:
Shall we unify these descriptions of `FMA` functions with the link of `Intel Intrinsics Guide`?
For example:
```
Multiply packed single-precision (32-bit) floating-point elements in a and b,
subtract packed elements in c from the intermediate result, and store the results in dst.
```